_episodes/02-benchmark-and-scaling.md

```diff
@@ -19,27 +19,244 @@ keypoints:
+Now we'll invoke LAMMPS executable to do the job. This can be done as following
+```
+srun lmp < in.lj
+```
+{: .input}
+
+Or, if you are using ```mpirun``` instead of ```srun```, the submission line would be:
+```
+mpirun -np 1 lmp < in.lj
+```
+{: .input}
+
+In this case, ```lmp``` is the name of the LAMMPS executable. But, in your HPC it could named something else.
+
+> ## Slurm script: full view
+>
+> Just following the exercise above can you create a batch file to submit a LAMMPS job for the above input file (say, in.lj) to 1 core only. You will submit the job to a partition/queue named ```batch```. The job is expected to take not more than 5 minutes, and the ```batch``` partition allows you to submit jobs not crossing 72 hours time limit. The name of the LAMMPS executable is ```lmp```. 
+>
+> > ## Solution
+> > ~~~
+> > #!/bin/bash -x
+> > {{ site.sched_comment }} --account=ecam
+> > {{ site.sched_comment }} --nodes=1
+> > {{ site.sched_comment }} --ntasks-per-node=1
+> > {{ site.sched_comment }} --output=mpi-out.%j
+> > {{ site.sched_comment }} --error=mpi-err.%j
+> > {{ site.sched_comment }} --time=72:00:00
+> > {{ site.sched_comment }} --partition=batch
+> >
+> > module use /usr/local/software/jureca/OtherStages
+> > module load Stages/Devel-2019a
+> > module load intel-para/2019a
+> > module load LAMMPS/9Jan2020-cuda
+> >
+> > srun lmp < in.lj > out.lj
+> > ~~~
+> > {: .input}
+> {: .solution}
+{: .challenge}
+
+## Understand the output files
+Let us now look for the output files. You would notice that in this case three files have been created: ```log.lammps```, ```mpi-out.xxxxx```, and ```mpi-err.xxxxx```. Among these three, ```mpi-out.xxxxx``` is mainly to capture the screen output that have been generated during the job execution. The purpose of the ```mpi-err.xxxxx``` file is to log entries if there is any error occurring during run-time. The one that is created by LAMMPS is called ```log.lammps```. 
+
+Once you open the ```log.lammps``` file you will notice that this file contains most of the important information starting from the LAMMPS version, number of processors used for the runs, processor lay out, thermdynamic steps, and the timing information. The header of a ```log.lammps``` file would be somewhat similar to this:
+```
+LAMMPS (18 Feb 2020)
+OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:94)
+  using 1 OpenMP thread(s) per MPI task
+```
+Note that it tells you about the LAMMPS version, and ```OMP_NUM_THREADS``` which is one of the important environment variables required to set for obtainging a performance boost. This would be discussed later in this tutorial. But for now, we'll focus mainly on the timing information provided in this file. 
+
+When the run concludes, LAMMPS prints the final thermodynamic state and a total run time for the simulation. It also appends statistics about the CPU time and storage requirements for the simulation. An example set of statistics is shown here:
+
+```
```

Comment:
As snippet of code from `_includes` maybe?

Reply:
Agree, this is better suited to a snippet so that it can be easily changed/updated.